### DISCUSSION TITLE
I have lisence key but reinstalled my pc thesedays...  and found I can not download video from the website which was ok in the past.

### DISCUSSION THREAD

**Discussion by lunux1818** [aclap-dev/video-downloadhelper#1385] [Help and Q&A]
### Your question

Can you pls help me w that problem ? thanks!
I don't know if it is ok to paste the license key here , lmk if you need that bro !
Kindly check the video link below ,
https://www.51cg1.com/archives/171197/

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: zh-CN
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"a7d2de5f22504423"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "F:\\idm",
    "concurrent_downloads_max": 10,
    "theme": "system",
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": true,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "forget_media_on_tab_close": true,
    "use_wide_ui": false,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 5,
    "record_download_history": true
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

For your license, just enter it again like you did the first time.  It's still valid.

For the video, here's a warning.  It's porn.  When I first visited the page, VDH offered me 2 variants, one of duration 1:40, the other of 8:37.  I tried to download the longer one with VDH but just got an error.  I didn't bother with the shorter one.  So I went to Plan B.  The Network Monitor did not show any master manifest, neither HLS not DASH.  But when I decided to filter the contents of the display using `m3u8` as my filter, I got 2 files.  Their type was `plain`.  That is utterly non-standard.  HLS manifests, both master & stream, are supposed to be of type x-mpegurl or vnd.apple.mpegurl or something a lot like one or the other of those.  Hiding a manifest in a file that is marked as plain is breaking the rules.  In any case, both m3u8 files turned out to be HLS stream manifests, so still no master manifest.  When I ran one of them through ffprobe, it told me the input was invalid.  But the other one gave me a proper report.  So I downloaded that with ffmpeg.  Here's the log from that download.

[Test mp4 Log.txt](https://github.com/user-attachments/files/17484849/Test.mp4.Log.txt)


I got quite good download speed, as you can see.  The resulting file played fine in VLC, although I only skimmed it.  I found I had to crank the volume up to what would normally be ear-splitting levels to hear any of the audio.  There didn't seem to be much audio anyway.  It looked like pretty low quality porn, as porn goes.

I would say that this site is so non-standard that you shouldn't expect any automated tool like VDH to work there.  You say VDH used to work on this site.  That is, apparently, ancient history.  Forget about what used to be.  VDH doesn't work there now, so I would put forward the hypothesis that the web site changed something.  I've never visited the site before so I can't compare what was with what is.

The whole page looks like either Japanese or Chinese.  Firefox didn't offer its usual translation button & I am not willing to go through the tedious exercise of copy/pasting it into Google Translate.  I have to say that at first glance, I don't see where there is a video player on the page.  I don't know how either VDH or the Network Monitor managed to find anything.  I also have no idea how many videos are on that page, nor where they are.  If you can read what it says there, more power to you.  Maybe you can work out some way of getting the content.

I have done everything I'm going to do here.

**Reply by GrampaWildWilly** [reply]
About pasting your license key in here . . .

**NEVER DO THAT!!!!!!**

Part of your license key is included in the `Copy information to clipboard` text that you pasted.  When I paste that information from my system, I remove even the partial license key.  There is utterly no use for even a partial license key in problem diagnosis.  EVER.  I don't know why that is included.  If I had been consulted about this before it was implemented, I would have strenuously objected to the idea.  Sadly, that never happened, so we're stuck with this.

If you post your license key in public, @jcfrog Jérôme generally cancels the key & sends you a new one.  You paid for the key.  Don't share it with the world.  You're supposed to keep it secret.

**Comment by lunux1818** [comment]
Hi GrampaWildWilly, many appreciated for you reply even you are the use same as me  <3 !
Well it seems I can not download any video from that website through VDH, but I found another tools to do it , let me share it with you here
Tool name is : N_m3u8DL-CLI_v2.9.9_with_ffmpeg_and_SimpleG
I can download video by pasting the url to this tool, seems work well!  
Thanks anyway my friend ,hope you have a good day!
